integration/querier_tenant_federation_test.go

```diff
@@ -151,7 +155,9 @@ func runQuerierTenantFederationTest(t *testing.T, cfg querierTenantFederationCon
 	result, err := c.Query("series_1", now)
 	require.NoError(t, err)
 
-	assert.Equal(t, mergeResults(tenantIDs, expectedVectors), result.(model.Vector))
+	// We don't guarantee series will be sorted in any particular order, so we ensure they're sorted consistently here
+	// to avoid assertion failures.
+	assert.Equal(t, sortVector(mergeResults(tenantIDs, expectedVectors)), sortVector(result.(model.Vector)))
```

Comment:
[nit] If we use `assert.ElementsMatch()` instead of `assert.Equal()`, would it avoid having to use `sortVector()` at all?

Reply:
Good point, fixed in 14ea31d.